src/main/java/mate/academy/rickandmorty/entity/Character.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.rickandmorty.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "characters")
+public class Character {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "external_id", unique = true)
+    private String externalId;
+
+    private String name;
```

Comment:
```suggestion
    @Column(nullable = false)
    private String name;
```

Add constraints for other fields